demo/runners/agent_container.py

```diff
@@ -96,8 +96,23 @@ def connection_ready(self):
         return self._connection_ready.done() and self._connection_ready.result()
 
     async def handle_oob_invitation(self, message):
+        print("handle_oob_invitation()")
         pass
 
+    async def handle_connection_reuse(self, message):
+        # we are reusing an existing connection, set our status to the existing connection
+        if not self._connection_ready.done():
+            self.connection_id = message["connection_id"]
+            self.log("Connected")
+            self._connection_ready.set_result(True)
+
+    async def handle_connection_reuse_accepted(self, message):
```

Comment:
Same question as above.

Reply:
This allows the controller (alice) to tag the active connection (handles the webhook)